### PR TITLE
feat: added skeleton instead of spinning loader for embeds

### LINKED ISSUE #13767: [CAL-3170] embed: replace spinner box with skeleton
image

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3170](https://linear.app/calcom/issue/CAL-3170/embed-replace-spinner-box-with-skeleton)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
- [ ] needs testing
- [ ] could not figure out how to test locally @hariombalhara 

here's the html sandbox https://play.tailwindcss.com/bpzsjZ5yBx?layout=horizontal

CleanShot 2024-02-19 at 21 25 52@2x

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -341,3 +341,4 @@ APP_ROUTER_TEAMS_ENABLED=0
 
 # disable setry server source maps
 SENTRY_DISABLE_SERVER_WEBPACK_PLUGIN=1
+NEXT_PUBLIC_HEAD_SCRIPTS=console.log('hey, Peer')
```
Comment (PeerRich) [inline .env.example:344]:
```suggestion
```

**Comment by ciaranha** [comment]
Looks great 👌

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
@PeerRich Getting this error "No loader element"

Screenshot 2024-02-20 at 5 39 53 PM

**Comment by Udit-takkar** [comment]
To test 

do cd packages/embeds/embed-core and yarn dev
and yarn dev on new terminal (default directory)

then go to http://localhost:3100/embed/

**Comment by PeerRich** [comment]
figured out some of the glitches but dont know how to load our custom colors in. thats why its a blank screen and also why dark mode is not working.


any ideas @Udit-takkar ?
https://github.com/calcom/cal.com/assets/8019099/2f1e22c2-f880-43d7-a2e8-deba62031811

**Comment by PeerRich** [comment]
added bg-gray-200 etc. instead of our token colors. should do the trick for now

**Review by Udit-takkar — COMMENTED** [review]
I also see Booker skeleton even when i open user page

https://github.com/calcom/cal.com/assets/53316345/e887b465-c953-4e13-a49d-e2ebcef56788

**Comment by Udit-takkar** [comment]
I think there is some issue in detecting theme on embed. even when you open embed of dark theme loader does not use color with dark: attribute

**Comment by PeerRich** [comment]
> I also see Booker skeleton even when i open user page

yeah I could not figure out yet how to differentiate between user profile, booking page and routing form. maybe @hariombalhara knows if this is possible?

**Comment by PeerRich** [comment]
we should probably fix this with atoms and give people proper react embeds and keep iframes as they are

**Pull request closed without merging**